azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -628,6 +635,16 @@ private void handleRemoveProject(final HttpServletRequest req,
       this.setErrorMessageInCookie(resp, e.getMessage());
       resp.sendRedirect(req.getRequestURI() + "?project=" + projectName);
       return;
+    } finally {
+      try {
+        //unschedule the project regardless
+        if (this.enableQuartz) {
```

Comment:
should this move to the removeProject try catch block?
Another question. Do we remove normal schedule today when removing a project?

Reply:
- Do we remove normal schedule today when removing a project
  Yes.

Reply:
- should this move to the removeProject try catch block?
Let me consolidate unschedule code together